velox/functions/prestosql/aggregates/tests/SumTest.cpp

```diff
@@ -461,5 +487,14 @@ TEST_F(SumTest, floatAggregateOverflow) {
   testAggregateOverflow<float, float, double>();
   testAggregateOverflow<double, double>();
 }
+
+TEST_F(SumTest, decimalAggregateOverflow) {
```

Comment:
Please skip this as it doesn't pass in fb internal test. I can help to turn on this later.